src/App.js

```diff
@@ -5,7 +5,11 @@ import LoginPage from './components/auth-components/login-page';
 import Lists from './components/Lists';
 import Items from './components/Items';
 import CreateShoppingList from './components/CreateShoppingList';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faIgloo } from '@fortawesome/free-solid-svg-icons';
 
+library.add(faIgloo);
```

Comment:
Was this just a test to see that FontAwesomeIcon was working? I can't see any other usage of `FontAwesomeIcon` in the PR?

Reply:
deleted